pkgs/development/python-modules/asn1ate/default.nix

```diff
@@ -3,13 +3,12 @@
 
 buildPythonPackage rec {
   pname = "asn1ate";
-  date = "20160810";
-  version = "unstable-${date}";
+  version= "0.6";
   name = "${pname}-${version}";
```

Comment:
You can delete the `name`.

Reply:
Isn't that mandatory?

Reply:
Not for Python. `buildPythonPackage` will automatically derive ot from `pname` and `version`.

Reply:
Done. Thanks for helping out!